src/driver.cpp

```diff
@@ -12,6 +12,31 @@ extern bpftrace::location loc;
 
 namespace bpftrace {
 
+namespace {
+std::string_view parse_license()
+{
+  std::string_view program = Log::get().get_source();
+  constexpr std::string_view spdx_id_marker = "SPDX-License-Identifier: ";
+  auto spdx_id_pos = program.find(spdx_id_marker);
```

Comment:
Should we ensure this is inside a comment? Seems like this would match this string anywhere in the file